linux/debian/man/km-config.1

```diff
@@ -0,0 +1,21 @@
+.\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.48.4.
```

Comment:
Is it possible to generate these at build time rather than storing in repo?

Reply:
Oops - yes! How did these files end up here? They do get generated...